### PR TITLE
Basic support for version 2 transactions. 

### PR DESCRIPTION
**Pull request by oscarguindzberg**
Cherry pick https://github.com/bitcoinj/bitcoinj/commit/850f21960723dfcb664f795404f3f42004bd32eb

### CODE REVIEW

File: core/src/main/java/org/bitcoinj/core/TransactionInput.java [hunk 1]
```diff
@@ -385,6 +385,14 @@ public boolean isOptInFullRBF() {
         return sequence < NO_SEQUENCE - 1;
     }
 
+    /**
+     * Returns whether this input, if it belongs to a version 2 (or higher) transaction, has
+     * <a href="https://github.com/bitcoin/bips/blob/master/bip-0068.mediawiki">relative lock-time</a> enabled.
+     */
+    public boolean hasRelativeLockTime() {
+        return (sequence & SEQUENCE_LOCKTIME_DISABLE_FLAG) == 0;
```
Comment (sqrrm) [inline core/src/main/java/org/bitcoinj/core/TransactionInput.java]: `SEQUENCE_LOCKTIME_DISABLE_FLAG` is not defined, was included in another commit with the same desciption in bitcoinj, 9e7cb1c951f630000b2d322e5bb2af827b597a10

Comment (oscarguindzberg) [reply core/src/main/java/org/bitcoinj/core/TransactionInput.java]: fixed.

**Comment by oscarguindzberg** [comment]
@sqrrm See https://github.com/bisq-network/bitcoinj/issues/17 regarding the jdk8/jdk10 issue when compiling bitcoinj.

**Comment by oscarguindzberg** [comment]
I just noticed the same fix is already ported to bitcoinj v0.14.7 https://github.com/bitcoinj/bitcoinj/commit/9e7cb1c951f630000b2d322e5bb2af827b597a10
I am working on a bisq's bitcoinj rebase from v0.14.7 so the fix would eventually get in.

**Pull request merged**